mobile/src/screens/chat/chat.tsx

```diff
@@ -49,7 +49,8 @@ const Chat: React.FC = () => {
   );
 
   const scrollViewReference = useRef<ScrollView | null>(null);
-  const messagesLength = currentChatMessages.length;
+  const messagesLength = Object.keys(currentChatMessages).length;
```

Comment:
```suggestion
  const messagesLength = Object.keys(currentChatMessages).flat().length;
```

I also changed that in my pr, cause before it was only count of days when messages were sent

Reply:
Thank you! I wasnt attentive